Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -35,9 +54,65 @@ internal static Uri EscapeUri(Uri uri)
         }
     }
 
+    public class BrowserLaunchOptions : IEquatable<BrowserLaunchOptions>
+    {
+        public BrowserLaunchOptions(Color? prefferedControlColor, Color? preferredBackgroundColor, Color? preferredTitleColor, BrowserLaunchMode launchMode, BrowserTitleMode shouldShowTitle)
```

Comment:
@jamesmontemagno may have an opinion, but I don't think we need this. Users can just use the object initializer syntax: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers

Reply:
Yeah, this is still a remnant from the readonly struct where every parameter can be assigned. I was unsure on whether to leave them in. I'll remove them